docusaurus.config.js

```diff
@@ -252,9 +252,33 @@ module.exports = {
             to: '/sdk/v3/guides/liquidity/modifying-position',
             from: ['/sdk/v3/guides/liquidity/adding', '/sdk/v3/guides/liquidity/removing'],
           },
+          // Test redirects
+          {
+            to: '/concepts/protocol/oracle',
+            from: [
+              '/protocol/concepts/V3-overview/oracle',
+              '/protocol/concepts/V3-overview/oracle?ref=hackernoon.com'
```

Comment:
```suggestion
```

docusaurus redirect doesnt like domains `hackernoon.com` so removed to fix build errors